src/uu/cut/src/cut.rs

```diff
@@ -14,11 +14,10 @@ use std::fs::File;
 use std::io::{stdin, stdout, BufRead, BufReader, Read, Stdout, Write};
```

Comment:
could you please fix the fmt issue? thanks

Reply:
fixed!